test/kafka_test.go

```diff
@@ -258,8 +256,8 @@ func (s *CLITestSuite) TestKafkaBroker() {
 
 		{args: "kafka broker get-tasks 1", fixture: "kafka/broker/get-tasks-1.golden"},
 		{args: "kafka broker get-tasks 1 --task-type remove-broker", fixture: "kafka/broker/get-tasks-1-remove-broker.golden"},
-		{args: "kafka broker get-tasks --all", fixture: "kafka/broker/get-tasks-all.golden"},
-		{args: "kafka broker get-tasks --all --task-type add-broker", fixture: "kafka/broker/get-tasks-all-add-broker.golden"},
+		{args: "kafka broker get-tasks", fixture: "kafka/broker/get-tasks-all.golden"},
```

Comment:
We're changing this to `kafka broker task list`, right?

Reply:
I couldn't find any tickets about this, so I went ahead and created one. I'll add it to this PR tomorrow.